### ISSUE TITLE
Plot overlay gives NaN error with uint8 images

### ISSUE THREAD

**Issue by rodriguez-facundo** [ANTsX/ANTsPy#60]
**Describe the bug**
I have a 3D image dtype='uint8' that I can plot like this:

`ants.plot(img)`

The image is segmented, so I can also plot different areas in isolation like this:

`ants.plot(img==3)` 

But when I try to overlay the images in a plot, I get:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-38-2d83e2fbdb38> in <module>
----> 1 ants.plot(img1, img2)

/opt/conda/lib/python3.7/site-packages/ants/viz/plot.py in plot(image, overlay, blend, alpha, cmap, overlay_cmap, overlay_alpha, cbar, cbar_length, cbar_dx, cbar_vertical, axis, nslices, slices, ncol, slice_buffer, black_bg, bg_thresh_quant, bg_val_quant, domain_image_map, crop, scale, reverse, title, title_fontsize, title_dx, title_dy, filename, dpi, figsize, reorient)
   1736                     overlay = overlay.reorient_image2('LAI')
   1737                 ov_arr = overlay.numpy()
-> 1738                 ov_arr[np.abs(ov_arr) == 0] = np.nan
   1739                 ov_arr = np.rollaxis(ov_arr, axis)
   1740 

ValueError: cannot convert float NaN to integer
```

**To Reproduce**
```
np1 = np.ones((3,3,3)).astype('uint8')
np2 = 2 * np1
img1 = ants.from_numpy(np1)
img2 = ants.from_numpy(np2)
ants.plot(img1, img2)
```

**Desktop (please complete the following information):**
 - mojave 10.14.4 | ubuntu 18.04
 - python 3.7

**Comment by rodriguez-facundo** [comment]
btw, I read about this line to overcome a weird NaN error from matplotlib with overlay in `ants/viz/plot.pt` but did not worked.

`warnings.simplefilter('ignore')`

**Comment by stnava** [comment]
thanks - I will take a look.   if you have a pull request ( a suggested fix to code ) that would be great.

**Comment by rodriguez-facundo** [comment]
Hi, is `ov_arr[np.abs(ov_arr) == 0] = np.nan` there to skip computation over positions with value NaN? (like using `np.nanmean`). NaN only exists in float type. My quick fix (🤣) would be:  

```
if overlat.dtype not in ['uint8', 'uint32']:
    ov_arr[np.abs(ov_arr) == 0] = np.nan
```

**Comment by stnava** [comment]
implemented your suggested fix here 5ed4d119242eeb6a61c4663389be83c9bac71ab5

please reopen if you have thought of some other approach
